provisioner/ansible/provisioner.go

```diff
@@ -111,6 +112,13 @@ func (p *Provisioner) Prepare(raws ...interface{}) error {
 		p.config.LocalPort = "0"
 	}
 
+	if p.config.User == "" {
+		p.config.User = os.ExpandEnv("${USER}")
```

Comment:
Also commented on the previous commit I think this needs to be %USERNAME% on Windows. I got a VM from here: https://dev.windows.com/en-us/microsoft-edge/tools/vms/mac/